### PR TITLE
Fix the resulting build url on Jira

### PR DESCRIPTION
**Pull request by godardt**
This build fixes the wrong url pasted on the Jira card. Previously the url of the latest build was pasted, now it paste the one associated with what was just uploaded

### CODE REVIEW

File: src/main/groovy/de/felixschulze/gradle/JiraTask.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.apache.http.client.HttpClient
+
+import java.nio.charset.Charset
+
+public class JiraTask {
+
+    static def String pasteBuildUrlToJiraCard(HttpClient httpClient, HockeyAppPluginExtension hockeyApp, String hockeyAppPublicUrl, String hockeyAppConfigUrl) {
+        def buildNumberUrl = hockeyAppPublicUrl + extractBuildNumberPath(hockeyAppConfigUrl)
+
+        if (hockeyApp.jiraUrlTitle != null &&
```
Comment (drommk) [inline src/main/groovy/de/felixschulze/gradle/JiraTask.groovy]: [see this ](https://paper.dropbox.com/doc/Android-Code-Style-RjwSLE62QeLItw89WbWfd#:uid=764255519&h2=Return-Early)

File: src/main/groovy/de/felixschulze/gradle/JiraTask.groovy [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            String base64EncodedCredentials = credentials.bytes.encodeBase64().toString()
+            httpPost.setHeader("Authorization", "Basic " + base64EncodedCredentials);
+            httpPost.setHeader(HTTP.CONTENT_TYPE, "application/json");
+
+            httpPost.setEntity(new ByteArrayEntity(data.getBytes(Charset.forName("UTF-8"))))
+            HttpResponse response = httpClient.execute(httpPost);
+
+            if (response.getStatusLine().getStatusCode() != HttpStatus.SC_CREATED) {
+                return "Error " + response.getEntity().getContent()
+            } else {
```
Comment (drommk) [inline src/main/groovy/de/felixschulze/gradle/JiraTask.groovy]: 🌷 🌷 🌷 this else is not mandatory

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ buildscript {
         maven { url "https://jitpack.io" }
     }
     dependencies {
-        classpath 'de.felixschulze.gradle:gradle-hockeyapp-plugin:3.6.4'
+        classpath 'com.github.heetch:hockeyapp-plugin:3.6.6'
```
Comment (drommk) [inline README.md:19]: is it not some kind of self-reference?

Comment (godardt) [inline README.md:19]: What do you mean? It's the readme file on how to install the plugin via Jitpack

Comment (drommk) [inline README.md:19]: ha ha shit nvm

**Pull request merged**